### DISCUSSION TITLE
[Docs] `aura conf --gen > ~/.config/aura/config.toml` fails because `~/.config/aura/` directory doesn't exist in the first place

### DISCUSSION THREAD

**Discussion by xfzv** [fosskers/aura#864] [Ideas]
A minor suggestion:

The [docs](https://fosskers.github.io/aura/install.html?search=aura%20conf%20--gen%20%3E%20~/.config/aura/config.toml) and the `README` mention the following command to generate a config file:

https://github.com/fosskers/aura/blob/5c40a89c5d4041ad67c465b48f0de1dedca98303/README.md?plain=1#L190-L195

However, this command will fail since `~/.config/aura` directory doesn't exist (unless the user already used `aura` in the past and kept the directory for some reason):

```console
% aura conf --gen > ~/.config/aura/config.toml
zsh: no such file or directory: /home/xfzv/.config/aura/config.toml

% aura conf --gen > ~/.config/aura/config.toml
bash: /home/xfzv/.config/aura/config.toml: No such file or directory
```

Maybe the directory creation could be automated? Or at least

```bash
mkdir -p ~/.config/aura
```

should be added to the documentation?

Keep up the good work!

**Comment by fosskers** [comment]
Yup, thanks. I will fix this.

**Reply by fosskers** [reply]
Done.
